pkgs/tools/security/vulnix/default.nix

```diff
@@ -19,6 +23,9 @@ python3Packages.buildPythonApplication rec {
     pytest-flake8
   ];
 
+  # indirect dependency ZConfig (from ZODB) does not support Python 3.8
+  disabled = python3Packages.pythonAtLeast "3.8";
+
```

Comment:
this shouldn't be necessary, instead, zconfig should be marked as disabled, and that will propagate to this package

Reply:
this will also cause the python38 version to fail, so this is completely uneeded
```
[07:46:36] jon@jon-desktop /home/jon/projects/nixpkgs (master)
$ nix-build -A python3Packages.zconfig
error: ZConfig-3.5.0 not supported for interpreter python3.8
```
```suggestion
```

Reply:
fixed